src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +50,208 @@ export const KafkaPage: FunctionComponent = () => {
             <CardHeader>
               <CardHeaderMain>
                 <CardTitle>
-                  <Title headingLevel="h3">
-                    {t("kafkaoverview:purchaseCardTitle")}
-                  </Title>
+                  <Title headingLevel="h3">{t("purchaseCardTitle")}</Title>
                 </CardTitle>
               </CardHeaderMain>
             </CardHeader>
-            <CardBody>{t("kafkaoverview:purchaseCardMainText")}</CardBody>
+            <CardBody>{t("purchaseCardMainText")}</CardBody>
             <CardFooter>
               <Button
                 data-testid="cardPurchase-buttonCTA"
-                variant={ButtonVariant.secondary}
+                variant={ButtonVariant.link}
+                isLarge
+                isInline
                 component="a"
                 href="https://marketplace.redhat.com/en-us/products/red-hat-openshift-streams-for-apache-kafka"
                 target="_blank"
               >
-                {t("kafkaoverview:purchaseCardCallToActionButton")}
+                {t("purchaseCardCallToActionButton")}
                 <ExternalLinkAltIcon className="pf-u-ml-md" />
               </Button>
             </CardFooter>
           </Card>
           <Card>
             <CardHeader>
               <CardTitle>
-                <Title headingLevel="h3">
-                  {t("kafkaoverview:contactSalesCardTitle")}
-                </Title>
+                <Title headingLevel="h3">{t("contactSalesCardTitle")}</Title>
```

Comment:
I see there's an `<h1>` element but not an `<h2>`. All of the card titles should be updated to use `headingLevel="h3"`